components/server/src/api/server.ts

```diff
@@ -208,39 +208,43 @@ export class API {
                         }
                     };
 
-                    const apply = async <T>(): Promise<T> => {
-                        const subjectId = await self.verify(context);
-                        await rateLimit(subjectId);
-                        context.user = await self.ensureFgaMigration(subjectId);
+                    return withRequestContext(async () => {
```

Comment:
does server streaming support `Promise<AsyncGenerator>`, I thought proxy should return AsyncGenerator?

Reply:
Uh, you are right: https://github.com/connectrpc/connect-es/blob/main/packages/connect/src/implementation.ts#L212C12-L212C12
Also, just makes sense... let me see how I can make that work...

Reply:
Done.